input/core/src/commonMain/kotlin/majestic/calendar/month/tools/MonthYearResources.kt

```diff
@@ -0,0 +1,9 @@
+package majestic.calendar.month.tools
+
+import androidx.compose.ui.graphics.vector.ImageVector
+
+data class MonthYearResources(
+    val previousMonth: ImageVector,
+    val nextMonth: ImageVector,
+    val dropDown: ImageVector
+)
```

Comment:
The class is already called `MonthYearResources`

So, instead of having properties called `previousMonth` or `nextMonth`, just the words `prev` and `next` are pretty straight forward


Also, it seems like your resources (at least for now) are just icons, so, why don't you just call this class `CalendarIcons`?